### PR TITLE
Add ppx_seq

### PR DESCRIPTION
**Pull request by hyphenrf**
This PR adds `ppx_seq.0.1`, a simple rewriter that just adds a Seq-literal syntax to OCaml programs.
If all is alright, please merge <3

### CODE REVIEW

**Comment by hyphenrf** [comment]
please don't merge now, I'll fix the CI issues and submit a new release

File: packages/ppx_seq/ppx_seq.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "Simple unintrusive ppx rewriter that offers Seq literals"
+maintainer: ["hyphens@pm.me"]
+authors: ["Hazem Elmasry"]
+license: "ISC"
+tags: ["syntax"]
+homepage: "https://github.com/hyphenrf/ppx_seq"
+bug-reports: "https://github.com/hyphenrf/ppx_seq/issues"
+depends: [
```
Comment (mseri) [inline packages/ppx_seq/ppx_seq.0.1/opam:9]:
```suggestion
depends: [
  "ocaml" { >= "4.07" }
```

This is where List.of_seq was added

**Comment by mseri** [comment]
You can make the fixes to the opam file in your repo without the need to reTag or rerelease, the metadata used by opam is the one from this repository. So if it is fixed here, it is enough

**Comment by hyphenrf** [comment]
@mseri might as well get it done upstream haha
and while I'm at it I got rid of that lint warning

**Comment by hyphenrf** [comment]
all checks passed. always good to see that shade of green :)

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**